packages/@aws-cdk/aws-pipes-alpha/lib/logs.ts

```diff
@@ -93,3 +142,81 @@ export interface ILogDestination {
    */
   grantPush(grantee: IRole): void;
 }
+
+/**
+ * CloudWatch Logs log group for delivery of pipe logs.
+ */
+export class CloudwatchLogsLogDestination implements ILogDestination {
+  private logGroup: ILogGroup;
+
+  constructor(logGroup: ILogGroup) {
+    this.logGroup = logGroup;
+  }
+
+  bind(_pipe: IPipe): LogDestinationConfig {
+    return {
+      parameters: {
+        cloudwatchLogsLogDestination: {
+          logGroupArn: this.logGroup.logGroupArn,
+        },
+      },
+    };
+  }
+
+  grantPush(pipeRole: IRole): void {
+    this.logGroup.grantWrite(pipeRole);
+  }
+}
+
+/**
+ * Firehose stream for delivery of pipe logs.
+ */
+export class FirehoseLogDestination implements ILogDestination {
+  private deliveryStream: IDeliveryStream;
+
+  constructor(deliveryStream: IDeliveryStream) {
+    this.deliveryStream = deliveryStream;
+  }
+
+  bind(_pipe: IPipe): LogDestinationConfig {
+    return {
+      parameters: {
+        firehoseLogDestination: {
+          deliveryStreamArn: this.deliveryStream.deliveryStreamArn,
+        },
+      },
+    };
+  }
+
+  grantPush(pipeRole: IRole): void {
+    this.deliveryStream.grantPutRecords(pipeRole);
+  }
+}
+
+/**
+ * S3 bucket for delivery of pipe logs.
+ */
+export class S3LogDestination implements ILogDestination {
+  private parameters: S3LogDestinationProps;
+
+  constructor(parameters: S3LogDestinationProps) {
+    this.parameters = parameters;
+  }
+
+  bind(_pipe: IPipe): LogDestinationConfig {
+    return {
+      parameters: {
+        s3LogDestination: {
+          bucketName: this.parameters.bucket.bucketName,
+          bucketOwner: this.parameters.bucket.env.account,
```

Comment:
I saw you're covering cross account bucket targets in your unit tests, might be worth making sure it's working correctly with an integ test as well

Reply:
This shouldn't matter though right?  The only difference with cross-account is that the bucket policy needs to be updated on the other side. Permissions on the role don't change, since there is no account ID in the arn.

Reply:
Wait, I think you're right.. you would need to use `fromBucketAttributes` and include `account` else CDK won't know which account it's from ?

Reply:
@nmussy Do you think it's safer if I add a `BucketOwner` property and default it to the account ID of the stack?

Reply:
Yeah I think this might be overly fancy, [ImportSourceSpecification](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_dynamodb.ImportSourceSpecification.html#bucketowner) just left the prop

Reply:
> and default it to the account ID of the stack?

I would just leave it empty, I can only imagine this becoming an issue, no advantages